price/priceService.ts

```diff
@@ -1,9 +1,23 @@
 import { baseUrl } from './url.js';
+import {Pair} from '../types/common.js'
 
-type Pair = string;
 type TimeFrame = string;
 
-async function fetchData(pair: Pair, timeFrame: TimeFrame): Promise<string> {
+interface candles {
```

Comment:
- Interfaces, by convention, are written in PascalCase
- This should be called `Candle`, not `Candles`, as it's pointing to one specific candle, not the whole array

Reply:
okay I fixed it